celery/bin/worker.py

```diff
@@ -109,6 +109,14 @@
     Maximum number of tasks a pool worker can execute before it's
     terminated and replaced by a new worker.
 
+.. cmdoption:: --maxmemkb
+
+    Maximum amount of resident memory that may be consumed by a
```

Comment:
Terminology here is a bit confusing, we usually reserve worker for 'celery worker instance', and `child process` for pool worker processes.